main.js

```diff
@@ -10,3 +10,18 @@
         
     }
 }())
+
+
+// Burger handler
+
+(function () {
+    const burgerItem = document.querySelector('.burger');
+    const menu = document.querySelector('.header_nav');
+    const menuCloseItem = document.querySelector('.header_nav-close');
+    burgerItem.addEventListener('click', () => {
+      menu.classList.add('header_nav_active');  
+    })
+    menuCloseItem.addEventListener('click', () => {
+        menu.classList.remove('header_nav_active');
+    });
```

Comment:
Question for you: can we replace add/remove with toggle? You can read about it [here](https://learn.javascript.ru/styles-and-classes) (just find it with "Find" via CTRL+F)

Reply:
In this case, there is no need to use toggle